qemu/tests/x86_cpu_flags.py

```diff
@@ -0,0 +1,47 @@
+import logging
+
+from avocado.utils import cpu
+from virttest import error_context, env_process
+from qemu.tests.x86_cpu_model import check_flags
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test cpu flags.
+    1) Check if current flags are in the supported lists, if no, cancel test
+    2) Otherwise, boot guest with the cpu flags
+    3) Check cpu flags in guest(only for linux guest)
+    4) Reboot guest
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    vendor_id = params.get("vendor_id", "")
+    if vendor_id:
+        if vendor_id != cpu.get_vendor():
+            test.cancel("This host vendor doesn't match the %s" % vendor_id)
```

Comment:
So it would be `This host vendor doesn't match the intel`, which sounds ill; recommend something like `The host vendor is not {vendor_id}, so it does not support this test case`.

Reply:
update and thanks.